cli/magic-config.ts

```diff
@@ -197,6 +202,67 @@ async function processCreateOptions(options: any): Promise<void> {
         return !(this as any).state.answers.enableSagemakerModels;
       },
     },
+    {
+      type: "confirm",
+      name: "enableSagemakerModelsSchedule",
+      message: "Do you want to enable a start/stop schedule for sagemaker models?",
+      initial: options.enableSagemakerModelsSchedule || false,
+      skip(): boolean {
+        return !(this as any).state.answers.enableSagemakerModels;
+      },
+    },
+    {
+      type: "input",
+      name: "sagemakerStartSchedule",
+      hint: "This cron format is using AWS eventbridge cron syntax see docs for more information",
+      message: "Start schedule for Sagmaker models expressed in UTC AWS cron format",
+      skip(): boolean {
+        return !(this as any).state.answers.enableSagemakerModelsSchedule;
+      },
+      validate(v: string) {
+        if ((this as any).skipped) {
+          return true
+        }
+        try {
+          AWSCronValidator.validate(v)
+          return true
+        }
+        catch (error) {
+          if (error instanceof Error){
```

Comment:
Would you trap and handle the `AWSCronError` separately here? If not, does that error class need to exist?